src/database/utilities/createRecord.js

```diff
@@ -341,6 +341,28 @@ const createTransactionItem = (database, transaction, item) => {
   return transactionItem;
 };
 
+const createProgramRequisition = (database, requisitionValues) => {
```

Comment:
can destructure like this (database, {period, currentUser, supplier, orderType})

Also, i am not sure if this needs to be another method. Can either: 
* Keep it as createProgramRequisition, but use createRequisition inside this method (to prevent repeating code)
* Or extend createRequisition: `const createRequisition = (database, user, otherStoreName, programSettings)`

With the second option:
```
const programValues = !programSettings ? {} : {
    otherStoreName: programSettings.supplier,
    daysToSupply: programSettings.orderType.maxMOS * 30,
    ...programSettings,
    orderType: programSettings.orderType.name,
}
const requisition = database.create('Requisition', {
  ...programValues
 ....
```

by the way, what extra comes from `...requisitionValues` ?

There is another option (my preferred), and that is: `const createRequisition = (database, user, otherStoreName, programSettings = {})`, and then make sure programSetting can be destructured directly into requisition

Reply:
I have this as one of the potential issues with the PR. Wasn't sure if I should fix this up here or make an issue afterward

Reply:
Hmm, since we are not particularly time pressed, i would suggest fixing in this PR

Reply:
Did this over a few commits
- removed the createProgramRequisition - 27787c22a42b37a8e1f3b3a16f992e202eb9e19b
- changed parameters - d05e85dde3ebfa2266291cd993a753f44c68ebdb